Library/Formula/openssl.rb

```diff
@@ -72,8 +66,8 @@ def install
         system "make", "clean"
       end
 
-      ENV.deparallelize
-      system "perl", "./Configure", *(configure_args + arch_args[arch])
+      # ENV.deparallelize
+      system "/usr/local/Cellar/perl/5.22.0/bin/perl", "./Configure", *(configure_args + arch_args[arch])
```

Comment:
```suggestion
      system "#{Formula["perl"].opt_bin}/perl", "./Configure", *(configure_args + arch_args[arch])
```
That said, I don't think this should be needed, as `brew` should already be putting Homebrew `perl` ahead of system `perl` in your `PATH` during the build.

Reply:
No, that didn't happen. It tried to use Perl 5.8. Remember this is Tigerbrew, not Homebrew

Reply:
I'm aware. I thought Tigerbrew also properly managed `PATH` during buildtime.